webpack.config.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const webpack = require('webpack');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+
+const license = fs.readFileSync(path.join(__dirname, 'LICENSE')).toString();
+const distDir = 'dist';
+
+module.exports = {
+  entry: ['babel-polyfill', './dist/metasync.js'],
+  output: {
+    path: path.join(__dirname, distDir),
+    filename: 'metasync.min.js',
+    sourceMapFilename: 'metasync.min.js.map',
+    library: 'metasync',
+  },
+  devtool: 'source-map',
+  plugins: [
+    new UglifyJsPlugin({
```

Comment:
Heads up: there's no `UglifyJsPlugin` in webpack 4 (webpack 3 is used in the PR). It's now built into webpack core and has its own field in the config, which one would rarely use directly as simply `mode: 'production'` enables both this and many other optimizations.